src/main/controllers/PlaceOfWorkController.ts

```diff
@@ -105,10 +107,10 @@ export default class PlaceOfWorkController {
     const { saveForLater } = req.body;
     if (saveForLater) {
       handleSessionErrors(req, res, this.form, PageUrls.CLAIM_SAVED);
-      handleUpdateDraftCase(req, this.logger);
+      handleUpdateDraftCase(req, logger);
     } else {
       handleSessionErrors(req, res, this.form, redirectUrl);
-      handleUpdateDraftCase(req, this.logger);
+      handleUpdateDraftCase(req, logger);
```

Comment:
I know this isnt a change you made but `handleUpdateDraftCase` is called on both branches, it could be moved outside this if statement

Reply:
I didn't even notice but good shout, will change

Reply:
What do you think to changing the lines above to:
`handleSessionErrors(req, res, this.form, req.body.saveForLater ? PageUrls.CLAIM_SAVED : redirectUrl);`